src/WebJobs.Script.Host/WebJobs.Script.Host.csproj

```diff
@@ -81,23 +81,23 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Azure.WebJobs.Extensions, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.2.0.0-alpha-10301\lib\net45\Microsoft.Azure.WebJobs.Extensions.dll</HintPath>
+      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.2.0.0-alpha-10307\lib\net45\Microsoft.Azure.WebJobs.Extensions.dll</HintPath>
```

Comment:
These package updates are temporary, referencing some Packages I generated locally and installed in my local nuget feed.

Before checking in, these updates will need to be staged. First I need to get the WebJobs.Script.Extensibility package up on MyGet, and these Extensions will move to the new model (I'll send a separate PR for that). We'll then absorb those new packages.